catalog/secrets/api/src/main/java/org/projectnessie/catalog/secrets/TokenSecret.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2024 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.catalog.secrets;
+
+import jakarta.annotation.Nonnull;
+import java.util.Map;
+
+/** Represents a single-value "token". */
+public interface TokenSecret extends Secret {
```

Comment:
Looks identical to `KeySecret`. Tokens are just a special-purpose "secret key".

Reply:
True - removed